apps/site/src/views/Home/sections/FAQ/FAQ.tsx

```diff
@@ -1,11 +1,52 @@
+import Image from "next/image";
 import Container from "react-bootstrap/Container";
+import Accordion from "react-bootstrap/Accordion";
+import styles from "./FAQ.module.scss";
+
+import star from "@/assets/images/star.svg";
+import eraser from "@/assets/images/eraser.svg";
+import { client } from "@/lib/sanity/client";
+
+interface sanityResponse {
```

Comment:
Nitpick [repeating]: Interfaces are usually title cased (i.e. `SanityResponse` and `FaqType`).